### PR TITLE
401 misc rename needs to support actions

### PR DESCRIPTION
**Pull request by rik86future**
Rename "Needs" in the UI and e2e tests, not controller or models

### CODE REVIEW

**Comment by rik86future** [comment]
[(misc) Rename Needs to Support Actions](https://trello.com/c/RcOnKyDr/401-misc-rename-needs-to-support-actions)

File: features/step_definitions/edit_support_actions_steps.rb [hunk 1]
```diff
@@ -1,54 +1,54 @@
-Given('a resident with a need exists') do
+Given('a resident with a support action exists') do
   @contact = Contact.create!(first_name: 'Test')
-  @need = Need.create!(contact_id: @contact.id, name: 'Phone Triage', category: 'Phone triage')
+  @need = Need.create!(contact_id: @contact.id, name: 'Dog walking', category: 'Dog walking')
```
Comment (timwis) [inline features/step_definitions/edit_support_actions_steps.rb:3]: Why was this changed to Dog walking? Was there an issue with the date field or something?
